.github/script/call-canary-deploy-demo-prod

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+const { spawnOrFail } = require('../../script/cli-utils');
+
+const myArgs = process.argv.splice(2);
+var demo_current_version = myArgs[0];
+
+var npm_latest_version = spawnOrFail('npm',['view', 'amazon-chime-sdk-js@latest', 'version']);
+var npm_beta_version = spawnOrFail('npm', ['view', 'amazon-chime-sdk-js@beta', 'version']);
+
+console.log('NPM latest version: ', npm_latest_version);
+console.log('NPM beta version: ', npm_beta_version);
+console.log('Demo current version: ', demo_current_version);
+
```

Comment:
I am using `console.log` here for logging. Is there any preference to use the logger from cli-utils? I can send a quick revision if needed.

Reply:
Using logger